src/shop-checkout.html

```diff
@@ -493,6 +496,12 @@ <h2 id="billAddressHeading">Billing Address</h2>
         }
       },
 
+      _cartChanged: function() {
+        if (this.cart && this.cart.length > 0) {
```

Comment:
nit: this check could be done in `shop-app` if you decide to handle `buy-cart` in that element.